README-template.md

```diff
@@ -764,130 +764,34 @@ the options, for example `spark.conf.set("temporaryGcsBucket", "some-bucket")` c
-   <td><strong><code>DoubleType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>NUMERIC</code></strong>
-   </td>
-   <td><strong><code>DecimalType</code></strong>
-   </td>
-   <td>
-     This preserves <code>NUMERIC</code>'s full 38 digits of precision and 9 digits of scope.
-   </td>
-  </tr>
-  <tr valign="top">
-     <td><strong><code>BIGNUMERIC</code></strong>
-     </td>
-     <td><strong><code>BigNumericUDT (UserDefinedType)</code></strong>
-     </td>
-     <td>
-       Scala/Java: BigNumericUDT DataType internally uses java.math.BigDecimal to hold the BigNumeric data.
-       <p> Python: BigNumericUDT DataType internally used python's Decimal class to hold the BigNumeric data.
-     </td>
-    </tr>
-  <tr valign="top">
-   <td><strong><code>STRING</code></strong>
-   </td>
-   <td><strong><code>StringType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>BYTES</code></strong>
-   </td>
-   <td><strong><code>BinaryType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>STRUCT</code></strong>
-   </td>
-   <td><strong><code>StructType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>ARRAY</code></strong>
-   </td>
-   <td><strong><code>ArrayType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>TIMESTAMP</code></strong>
-   </td>
-   <td><strong><code>TimestampType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>DATE</code></strong>
-   </td>
-   <td><strong><code>DateType</code></strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>DATETIME</code></strong>
-   </td>
-   <td><strong><code>StringType</code></strong>
-   </td>
-   <td>Spark has no DATETIME type. Casting to TIMESTAMP uses a configured TimeZone, which defaults to the local timezone (UTC in GCE / Dataproc).
-<p>
-We are considering adding an optional TimeZone property to allow automatically  converting to TimeStamp, this would be consistent with Spark's handling of CSV/JSON (except they always try to convert when inferring schema, and default to the local timezone)
-   </td>
-  </tr>
-  <tr valign="top">
-   <td><strong><code>TIME</code></strong>
-   </td>
-   <td><strong><code>LongType</code></strong>
-   </td>
-   <td>Spark has no TIME type. The generated longs, which indicate <a href="https://avro.apache.org/docs/1.8.0/spec.html#Time+%2528microsecond+precision%2529">microseconds since midnight</a> can be safely cast to TimestampType, but this causes the date to be inferred as the current day. Thus times are left as longs and user can cast if they like.
-<p>
-When casting to Timestamp TIME have the same TimeZone issues as DATETIME
-   </td>
-  </tr>
-</table>
+| **BigQuery Standard SQL Data Type**  | **Spark SQL Data Type**               | **Notes**       |
```

Comment:
I feel it is more it is more intuitive to have the notes in the column next to types rather than having it below.
Also, if we are having the notes below, we can delete the column `Notes`, since there it does not have any content in it anyway.

Reply:
Given the goal of sticking to markdown structures over html, I think it makes more sense to keep the bottom notes and remove the notes column altogether. While I would appreciate a separate column for these notes, the notes themselves are too long to reasonably fit into a column. I'll correct this.